Makefile

```diff
@@ -47,6 +47,18 @@ endif
 .PHONY: all doc
 all: install
 
+GO_DOWNLOAD:=https://storage.googleapis.com/golang
+GO_VERSION:=1.9.2
+GO_OS:=linux
+GO_ARCH:=amd64
+GO_PKG:=go$(GO_VERSION).$(GO_OS)-$(GO_ARCH).tar.gz
+
+.PHONY: golang
+golang:
```

Comment:
I would prefer to add golang installation things. This is not the goal of Skydive to install proper Go environment. I know that lot of us have different GOROOT environment.

Reply:
Will remove `golang` target